src/styles/components/input.scss

```diff
@@ -85,18 +85,21 @@
 }
 
 .miter-input__label {
-  font-size: 11px;
-  justify-content: center;
-  line-height: 14px;
-  margin-bottom: 4px;
-  opacity: 80%;
-  padding-left: 2px;
+  font-size: 13px;
+  font-family: 'input-sans';
```

Comment:
We should use the sass variable for this (and the other place in this file) `$font--secondary`

Reply:
I tried, I get this even though
<img width="1464" alt="Screen Shot 2022-02-22 at 2 35 02 PM" src="https://user-images.githubusercontent.com/40467112/155206021-c9ae7bb0-e187-4002-be05-ebc1de52a518.png">
 the @use of the _variables is implemented at the top of the file

Reply:
You'll need to use `variables.$font--secondary`, since those `@use` imports don't have unwrappers like:
```
@use '../abstracts/variables' as *;
```

Reply:
Gotcha